public/index.html

```diff
@@ -43,6 +43,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g==" crossorigin="anonymous" referrerpolicy="no-referrer" />
```

Comment:
Can we avoid font awesome or any other CSS library and use as much CSS by hand as possible? It is to make sure, we are not opinionated about fontawesome vs bootstrap vs material etc. Maybe in future, we will take a call as a community and get one.

Also for the icons, react-icons library is already in to make use of it.

Reply:
Done!